ext_mongodb.php

```diff
@@ -620,6 +626,94 @@ class BulkWriteException extends WriteException {}
 /* {{{ BSON and Serialization Classes */
 namespace MongoDB\BSON;
 
+/* {{{ Interfaces */
+interface BinaryInterface
+{
+	public function serialize() : string;
+	public function unserialize(mixed $data) : void;
```

Comment:
Any reason for explicitly listing these methods instead of extending [Serializable](http://php.net/serializable)?

More importantly: If these interfaces are intended to be common to PHPC, which uses `__wakeup()`, I don't think we can include these methods in the interface.